### PR TITLE
Fix BrowserPanel#displayReader test

### LINKED ISSUE #97: Testing: Remove remote resources from test files for BrowserPanelTest#displayReader
Remove remote resources from test files for `BrowserPanelTest#displayReader` as remote resources are not needed in order to unit test the reader view and also cause the test to fail because of timeouts.

### PR DESCRIPTION
**Pull request by rlrh**
Resolves #97 
#### Summary of changes:
- Add new local placeholder image used to replace remote images
- Remove all remote resources (but not links) from Wikipedia test page
- Add new HTML5 test page without remote resources
- Add new entry in `TypicalEntries` for new test page
- Refactor `BrowserPanelTest#displayReader` and add new test for reader view on new test entry

### CODE REVIEW

File: src/test/resources/view/BrowserPanelTest/test.html [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+                <button type="reset">&lt;button type=reset&gt;</button>
+                <button type="button" disabled>&lt;button disabled&gt;</button>
+              </p>
+            </fieldset>
+            <p><a href="#top">[Top]</a></p>
+          </form>
+        </section>
+      </main>
+      <footer role="contentinfo">
+        <p>Made by <a href="http://twitter.com/cbracco">@cbracco</a>. Code on <a href="http://github.com/cbracco/html5-test-page">GitHub</a>.</p>
```
Comment (thomastanck) [inline src/test/resources/view/BrowserPanelTest/test.html:447]: I'm not sure what we need to do for MIT licensed resources like these. Maybe to be safe we should copy their license text along with a link to their repo into this HTML file as a comment at the top of the file?

**Review by thomastanck — COMMENTED** [review]
Good fix. I would try to minimise the diff on the HTML page but I think the autoformatter kicked in.

**Review by epicfailname — APPROVED** [review]
LGTM

File: src/test/resources/view/BrowserPanelTest/test.html (hunk 1 above)
Comment (rlrh) [reply src/test/resources/view/BrowserPanelTest/test.html:447]: Thanks for the fix

**Pull request merged**